Common/brushed_dc.hpp

```diff
@@ -0,0 +1,25 @@
+#pragma once
+
+namespace sjsu
+{
+    class brushed_dc
```

Comment:
```suggestion
  class brushed_dc
```
Please make tabs 2 spaces.